app/src/features/projects/view/components/ProjectObjectives.test.tsx

```diff
@@ -36,42 +49,64 @@ describe('ProjectObjectives', () => {
 
   it('renders correctly when objectives are in multiple paragraphs', () => {
     const multilineObjectives = 'Paragraph1\nParagraph2\n\nParagraph3';
+    const multilineCaveats = 'Paragraph1\nParagraph2\n\nParagraph3';
 
     const { asFragment } = render(
       <ProjectObjectives
         projectWithDetailsData={{
           ...projectWithDetailsData,
-          objectives: { ...projectWithDetailsData.objectives, objectives: multilineObjectives }
+          objectives: {
+            ...projectWithDetailsData.objectives,
+            objectives: multilineObjectives,
+            caveats: multilineCaveats
+          }
         }}
       />
     );
 
     expect(asFragment()).toMatchSnapshot();
   });
 
-  it('allows to read more to enable viewing the entire objectives string and read less to only show the truncated string', () => {
+  it('the read more and read less buttons are working', async () => {
```

Comment:
nit so that the test sentence reads well:
```suggestion
  it('functions as expected with the read more and read less buttons', async () => {
```

Reply:
also we shouldnt need the async in this function here

Reply:
done